packages/@lightningjs/ui-components/src/components/ControlRow/ControlRow.d.ts

```diff
@@ -16,34 +16,97 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import type lng from '@lightningjs/core';
-import type { StylePartial } from '../../types/lui';
+import lng from '@lightningjs/core';
+import { StylePartial } from '../../types/lui';
 import TitleRow, { TitleRowStyle } from '../TitleRow';
 
-export type ControlRowStyle = TitleRowStyle & {
+type ControlRowItem = {
+  extraItemSpacing?: number;
+};
```

Comment:
why was this changed? this isn't a valid component type, and it doesn't seem like `extraItemSpacing` is a valid property on either controls or items